src/guide/get-started/launch-iroha.md

```diff
@@ -0,0 +1,55 @@
+# Launch Iroha
+
+1. Install the prerequisites:
+
+   - [Docker](https://docs.docker.com/get-docker/)
+   - [Docker compose](https://docs.docker.com/compose/)
+
+2. [Install Iroha](./install-iroha.md).
+
+3. Navigate to the repository root directory.
+ 
+4. Run `docker compose` and specify the network configuration file to bring up a network of 4 containerized peers:
+
+   ```bash
+   $ docker compose -f defaults/docker-compose.yml up
+   ```
+
+   Depending on your set-up, this might either
```

Comment:
Would be good to link `iroha_swarm` readme here for setup

Reply:
linked it in the Docker Options section a bit lower